docs/book/src/tasks/certs/auto-rotate-certificates-in-kcp.md

```diff
@@ -0,0 +1,59 @@
+## Automatically rotating certificates using Kubeadm Control Plane provider
+
+When using Kubeadm Control Plane provider (KCP) it is possible to configure automatic certificate rotations. KCP does this by triggering a rollout when the certificates on the machines are about to expire.
```

Comment:
Let's add a note explaining that this is automatically enabled only for KCP-managed machines created with CAPI >= v1.3; also add a paragraph about how to set this for existing machines (ssh into the machine, run kubeadm command to get certificate expiration date, set the annotation on the machine) and reference it from here.